HomeWork/src/main/java/arraylist/TasksWithList.java

```diff
@@ -55,6 +58,7 @@ static public List<String> calcOccurrence(List<String> elements) {
     }
 
     public int countOccurrence(List<String> elements, String value) {
+        Objects.requireNonNull(elements);
         Objects.requireNonNull(value);
         int counter = 0;
         for (int i = 0; i < elements.size(); i++) {
```

Comment:
If you wanna compare 2 strings, use `equals`. Why are you using `contains`?  It is incorrect.